client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
+import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
+import { useInterval } from 'calypso/lib/interval';
+import type { Step } from '../../types';
+import 'calypso/assets/stylesheets/shared/_loading.scss';
+import './style.scss';
+
+const useSteps = () => {
```

Comment:
Can we use `loadingMessages` or anything you like other than steps? They can be easily confused with flow steps.